app/browser/menu.js

```diff
@@ -266,6 +266,7 @@ const createViewSubmenu = () => {
       label: locale.translation('toggleDeveloperTools'),
       accelerator: isDarwin ? 'Cmd+Alt+I' : 'Ctrl+Shift+I',
       click: function () {
+        const appStore = require('../../js/stores/appStore')
```

Comment:
This was already fixed, so if you rebase to the master, you can remove this code :wink: